<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>248</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>296</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: rgb(192, 192, 192);
	color: white;
	border: 1px solid gray;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_calculator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(0, 170, 255);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0,x2: 0, y2:1,
                              stop: 0 #00BFFF, stop: 1 #1E90FF);
}</string>
    </property>
    <property name="text">
     <string>Калькулятор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trafficAnalysis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(0, 170, 255);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0,x2: 0, y2:1,
                              stop: 0 #00BFFF, stop: 1 #1E90FF);
}</string>
    </property>
    <property name="text">
     <string>Анализ трафика</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
	background-color: rgb(192, 192, 192);
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_calculator"/>
    <addaction name="action_trafficAnalysis"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>Отчистить экран</string>
   </property>
  </action>
  <action name="action_calculator">
   <property name="text">
    <string>Калькулятор</string>
   </property>
  </action>
  <action name="action_trafficAnalysis">
   <property name="text">
    <string>Анализатор трафика</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
